Description: More utempter path searching, this one to keep Debian stable happy
Author: Jonathan Riddell
Origin: upstream
Forwarded: not-needed
Applied-Upstream: http://commits.kde.org/kpty/053ee8add12db44e6eaa68fd02e34ef1b97ba528
Last-Update: 2016-11-28
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/cmake/FindUTEMPTER.cmake
+++ b/cmake/FindUTEMPTER.cmake
@@ -37,8 +37,8 @@
 find_file (UTEMPTER_EXECUTABLE utempter PATHS
     ${KDE_INSTALL_FULL_LIBEXECDIR}/utempter
     ${KDE_INSTALL_FULL_LIBDIR}/utempter
-    ${CMAKE_PREFIX_PATH}/libexec/utempter
-    ${CMAKE_PREFIX_PATH}/lib/utempter
+    ${CMAKE_INSTALL_PREFIX}/libexec/utempter
+    ${CMAKE_INSTALL_PREFIX}/lib/utempter
     /usr/libexec/utempter
 )
 
